### PR TITLE
[WIP] Fix broken link in robsense telemetry doc

### PR DESCRIPTION
**Pull request by hamishwillee**
There is still a broken link to easyswarm - anyone we can ask about that?

### CODE REVIEW

File: common/source/docs/common-telemetry-robsense-swarmlink.rst [hunk 1]
```diff
@@ -24,12 +25,13 @@ Specifications (according to the manufacturer)
...
 - Weight: 65g
 
-More details can be found `here <http://www.robsense.com/swarmlink-en.html>`__.
+More details can be found `here <http://en.robsense.com/prod_view.aspx?TypeId=69&Id=177&FId=t3:69:3>`__.
 
 EasySwarm
 ---------
 
-Built on top of the SwarmLink hardware, `EasySwarm <http://www.robsense.com/easyswarm-en.html>`__ is a developer focused ground station and development kit aimed at making swarming easier.  This free and open source software can be found in the `RobSense SwarmLink github repo <https://github.com/RobSenseTech/SwarmLink>`__.
+Built on top of the SwarmLink hardware, `EasySwarm <http://www.robsense.com/easyswarm-en.html>`__ is a developer focused ground station and development kit aimed at making swarming easier.
```
Comment (hamishwillee) [inline common/source/docs/common-telemetry-robsense-swarmlink.rst]: This link is broken. The github link below still exists, but concerned that if this one is gone, perhaps the repo is no longer properly maintained. Anyone we can ask?

Comment (auturgy) [reply common/source/docs/common-telemetry-robsense-swarmlink.rst]: I can email Jay at Robsense, but @zb314e might know

Comment (auturgy) [reply common/source/docs/common-telemetry-robsense-swarmlink.rst]: @jiangfengping would you be able to provide an update: would be good to get this page up to date.
Thanks!

Comment (rmackay9) [reply common/source/docs/common-telemetry-robsense-swarmlink.rst]: The EasySwarm link we should use is this: http://en.robsense.com/prod_view8.aspx?TypeId=69&Id=177&FId=t3:69:3

**Comment by Hwurzburg** [comment]
updated links, kept github link...closing

**Pull request closed without merging**